packages/database/package.json

```diff
@@ -56,7 +56,8 @@
     "@firebase/app": "0.7.25",
     "rollup": "2.72.1",
     "rollup-plugin-typescript2": "0.31.2",
-    "typescript": "4.2.2"
+    "typescript": "4.2.2",
+    "uuid": "^8.3.2"
```

Comment:
Can we avoid bringing in an extra dep here? I see in app-check we just implemented a function ourselves, is this good enough? https://github.com/firebase/firebase-js-sdk/blob/6f0049e66064809ae990a2d9461e28b2d6d08d19/packages/app-check/src/util.ts#L43

If it is, maybe we can extract it into `@firebase/util` and share it between app-check and database (maybe in a later PR, just have a duplicate implementation in this one?)

Reply:
Yeah, let me just reimplement it here and add a TODO